FAILED api/tests/unit_tests/services/test_summary_index_service.py::test_vectorize_summary_created_record_becomes_none_triggers_guard - AttributeError: 'NoneType' object has no attribute 'id'

@dosu how to fix

--- a/api/services/summary_index_service.py
+++ b/api/services/summary_index_service.py
@@ -348,6 +348,8 @@ class SummaryIndexService:
                                     status=SummaryStatus.COMPLETED,
                                     enabled=True,
                                 )
+                                if summary_record_in_session is None:
+                                    raise RuntimeError("summary_record_in_session should not be None at this point")
                                 summary_record_in_session.id = summary_record_id
                                 session.add(summary_record_in_session)
                                 logger.info(

